website/patch-guides.sh

```diff
@@ -0,0 +1 @@
+ln -s documentation/guides/tutorials/make-a-zio-app-configurable/docs/make-a-zio-application-configurable.md website/docs/guides/tutorials/
```

Comment:
It appears that something has gone wrong with the build file that we require this symlink.

Reply:
Yes, that specific guide in documentation folder broke and wasn’t detected during sidebar generation step

I did try looking at the build.sbt file but I’m not really familiar with Scala lang and it’s tooling.

if you have any suggestions for this I’d be happy to look into it.

Reply:
Did this happen before or after you updated your work to align with the series/2.x branch?